src/styles/components/header.scss

```diff
@@ -20,6 +18,13 @@ $nav-bar-link: var(--nav-bar-link);
 	padding: 0.5rem 1rem;
 	background-color: $header-bg;
 
+	.mobile-logo-position {
```

Comment:
@Aringoaway The logo looks to me a bit shifted to the right. Is there any option how to center it without using too much styling as you have below? What I mean something like `text-align: -webkit-center` maybe? But I am just guessing about that suggested option...